util/site-dashboard/ot-nightly-results.js

```diff
@@ -0,0 +1,79 @@
+  ["aon_timer", ["https://reports.opentitan.org/hw/ip/aon_timer/dv/latest/"]],
+  ["clkmgr", ["https://reports.opentitan.org/hw/ip/clkmgr/dv/latest/"]],
+  ["csrng", ["https://reports.opentitan.org/hw/ip/csrng/dv/latest/"]],
+  ["edn", ["https://reports.opentitan.org/hw/ip/edn/dv/latest/"]],
+  ["entropy_src", ["https://reports.opentitan.org/hw/ip/entropy_src/dv/latest/"]],
+  ["flash_ctrl", ["https://reports.opentitan.org/hw/ip/flash_ctrl/dv/latest/"]],
+  ["gpio", ["https://reports.opentitan.org/hw/ip/gpio/dv/latest/"]],
+  ["hmac", ["https://reports.opentitan.org/hw/ip/hmac/dv/latest/"]],
+  ["i2c", ["https://reports.opentitan.org/hw/ip/i2c/dv/latest/"]],
+  ["keymgr", ["https://reports.opentitan.org/hw/ip/keymgr/dv/latest/"]],
+  ["kmac", ["https://reports.opentitan.org/hw/ip/kmac_masked/dv/latest/", "https://reports.opentitan.org/hw/ip/kmac_unmasked/dv/latest/"]],
+  ["lc_ctrl", ["https://reports.opentitan.org/hw/ip/lc_ctrl/dv/latest/"]],
+  ["otbn", ["https://reports.opentitan.org/hw/ip/otbn/dv/uvm/latest/"]],
+  ["otp_ctrl", ["https://reports.opentitan.org/hw/ip/otp_ctrl/dv/latest/"]],
+  ["pattgen", ["https://reports.opentitan.org/hw/ip/pattgen/dv/latest/"]],
+  ["prim_alert", ["https://reports.opentitan.org/hw/ip/prim/dv/prim_alert/latest/"]],
+  ["prim_esc", ["https://reports.opentitan.org/hw/ip/prim/dv/prim_esc/latest/"]],
+  ["prim_lfsr", ["https://reports.opentitan.org/hw/ip/prim/dv/prim_lfsr/latest/"]],
+  ["prim_present", ["https://reports.opentitan.org/hw/ip/prim/dv/prim_present/latest/"]],
+  ["prim_prince", ["https://reports.opentitan.org/hw/ip/prim/dv/prim_prince/latest/"]],
+  ["pwm", ["https://reports.opentitan.org/hw/ip/pwm/dv/latest/"]],
+  ["pwrmgr", ["https://reports.opentitan.org/hw/ip/pwrmgr/dv/latest/"]],
+  ["rom_ctrl", ["https://reports.opentitan.org/hw/ip/rom_ctrl/dv/latest/"]],
+  ["rstmgr", ["https://reports.opentitan.org/hw/ip/rstmgr/dv/latest/"]],
+  ["rstmgr_cnsty_chk", ["https://reports.opentitan.org/hw/ip/rstmgr/dv/rstmgr_cnsty_chk/latest/"]],
+  ["rv_dm", ["https://reports.opentitan.org/hw/ip/rv_dm/dv/latest/"]],
+  ["rv_timer", ["https://reports.opentitan.org/hw/ip/rv_timer/dv/latest/"]],
+  ["spi_device", ["https://reports.opentitan.org/hw/ip/spi_device/dv/latest/"]],
+  ["spi_host", ["https://reports.opentitan.org/hw/ip/spi_host/dv/latest/"]],
+  ["sram_ctrl", ["https://reports.opentitan.org/hw/ip/sram_ctrl_main/dv/latest/", "https://reports.opentitan.org/hw/ip/sram_ctrl_ret/dv/latest/"]],
+  ["sysrst_ctrl", ["https://reports.opentitan.org/hw/ip/sysrst_ctrl/dv/latest/"]],
+  ["tl_agent", ["https://reports.opentitan.org/hw/dv/sv/tl_agent/dv/latest/"]],
+  ["uart", ["https://reports.opentitan.org/hw/ip/uart/dv/latest/"]],
+  ["usbdev", ["https://reports.opentitan.org/hw/ip/usbdev/dv/latest/"]],
+  ["xbar", ["https://reports.opentitan.org/hw/top_earlgrey/ip/xbar_main/dv/autogen/latest/", "https://reports.opentitan.org/hw/top_earlgrey/ip/xbar_peri/dv/autogen/latest/"]],
```

Comment:
Does this list get populated automatically? If not, I suggest tracking this as an enhancement in a new issue

Reply:
It doesn't, it's hand written as there's a few subtleties in which URL is used. It will be further complicated by the addition of Ibex (which will have its reports at ibex.reports.lowrisc.org).

But yes auto generation certainly feasible and a good future improvement.

Reply:
@hcallahan-lowrisc: Please create an issue to track this